api/contrib/clave/clave.php

```diff
@@ -32,7 +32,7 @@
  * Tambien se puede llamar desde un metodo de la siguiente manera:
  * modules_loader("clave");       <-- Esta funcion importa el modulo
  * getClave($tipoDocumento="",$tipoCedula = "", $cedula = "", $situacion = "", $codigoPais = "", $consecutivo = "", $codigoSeguridad = "")  <------------ esta funcion retorna la clave
- * */
+ **/
```

Comment:
Sugerencia sin mayor importancia, no creo que afecte a phpdoc pero el resto de los comentarios cierran así:

```suggestion
 */
```

Reply:
fixed.